orders/src/main/resources/static/css/orders.css

```diff
@@ -0,0 +1,58 @@
+.tg {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+.tg td {
+    font-family: David, sans-serif;
+    font-size: 14px;
```

Comment:
use relative size instead of pixel size. See https://www.w3schools.com/cssref/css_units.asp

Reply:
fine